src/Middleware/HttpLogging/src/HttpLoggingExtensions.cs

```diff
@@ -20,8 +20,8 @@ internal static partial class HttpLoggingExtensions
         exception: null,
         formatter: HttpResponseLog.Callback);
 
-    [LoggerMessage(3, LogLevel.Information, "RequestBody: {Body}", EventName = "RequestBody")]
-    public static partial void RequestBody(this ILogger logger, string body);
+    [LoggerMessage(3, LogLevel.Information, "RequestBody: {Body}{status}", EventName = "RequestBody")]
```

Comment:
Logging tokens should always be pascal case, e.g. `{Status}`.

Is the formatting right here? Should there be a space between body and status?

Reply:
The lack of space was intentional, the status is going to be something like [Truncated], and it will be empty in the common case.